packages/nextjs/services/web3/wagmi-burner/burnerWalletConfig.ts

```diff
@@ -20,8 +25,17 @@ export const burnerWalletConfig = ({ chains }: BurnerWalletOptions): Wallet => (
   name: burnerWalletName,
   iconUrl: "https://avatars.githubusercontent.com/u/56928858?s=200&v=4",
   iconBackground: "#0c2f78",
-  //todo add conditions to hide burner wallet
-  // hidden: () => true,
+  hidden: () => {
+    if (!burnerConfig.enabled) {
```

Comment:
What should the default behavior be (i.e. when enabled is not defined)?

If we want to enable it by default, maybe we should put

```typescript
if (burnerConfig.enabled === false) {
  return true;
}
```

Then an undefined setting will fall through to the rest of the check and eventually default to not be hidden.

Reply:
True. There's some workarounds (such as casting to `ScaffoldConfig` in places where we don't need the narrow typing), but probably not worth it while our config is still relatively simple.